src/blocks/dynamic-non-interactive-parent/block.json

```diff
@@ -0,0 +1,33 @@
+{
+	"apiVersion": 2,
+	"title": "BHE Alpine - Dynamic Non Interactive Parent",
+	"name": "bhe-alpine/dynamic-non-interactive-parent",
+	"category": "text",
+	"icon": "flag",
+	"usesContext": [
+		"postId",
+		"postType",
+		"queryId"
+	],
+	"attributes": {
+		"title": {
+			"type": "string",
+			"source": "text",
+			"selector": "h2",
+			"public": true,
```

Comment:
These `public` fields are not necessary with Alpine, are they?

Reply:
You're right! Copy & Pasting issues 😄 I've just changed it